src/plugins/wizard/public/application/utils/state_management/store.ts

```diff
@@ -35,3 +35,4 @@ export type AppDispatch = Store['dispatch'];
 
 export { setState as setStyleState, StyleState } from './style_slice';
 export { setState as setVisualizationState, VisualizationState } from './visualization_slice';
+export { setState as setMetadataState, MetadataState } from './metadata_slice';
```

Comment:
Do we still need this export?

Reply:
yea we do not need this since i change the implementation. Good catch